models/articles.models.js

```diff
@@ -7,15 +7,23 @@ exports.fetchTopics = () => {
 };
 
 exports.fetchArticleById = (article_id) => {
-    return db.query("SELECT * FROM articles WHERE article_id = $1", [article_id])
-    .then(({rows}) => {
-        if(!rows.length) {
+ return Promise.all([
```

Comment:
Would like you to refactor this into a single PSQL query. Join the comments table on to the reviews and use COUNT() to get this all in one query, rather than doubling the amount of queries and time it takes for a query.